plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -893,14 +894,36 @@ $.extend(DataTable.prototype, UIControl.prototype, {
             $searchAction.addClass('searchActive forceActionVisible');
             var width = getOptimalWidthForSearchField();
```

Comment:
This part seems to behave weird on same tables, and different to the "single control count" overall.

Inside `getOptimalWidthForSearchField` the first `.dataTableControls` element is used, so most likely the top one. And that is then used to determine the size of the search field.

Both the "Page titles" and "Page URLs" report on "Behaviour > Performance" are displaying an 80px width search field on the top, and 122px on the bottom. On "demo" this both bottom search inputs are receiving a 250px width. And "Behaviour > Pages" went from 250px down to just 158px.

I think the reason are the missing `s9 m9` classes on the header variant of `col dataTableControls`, just adding those could already solve the issue for all instances.